pysph/base/octree_nnps.pxd

```diff
@@ -21,7 +21,7 @@ cdef class OctreeNNPS(NNPS):
     ##########################################################################
     # Data Attributes
     ##########################################################################
-    cdef public list tree
+    cdef list tree
```

Comment:
Just curious, why have you made this non-public?

Reply:
I had actually had to enable this for self.method hack. So, I re-turned it to non-public state now.